package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@holochain/lair-client",
+  "version": "0.1.0",
+  "description": "Javascript implementation of client for Lair Keystore",
+  "main": "src/index.js",
+  "module": "src/index.js",
+  "scripts": {
+    "test": "make test-debug",
+    "test-unit": "make test-unit-debug",
+    "test-integration": "make test-integration-debug"
+  },
+  "author": "Holochain",
+  "contributors": [
+    "Matthew Brisebois <matthew.brisebois@holo.host>",
+    "Zippy <eric@holo.host>"
+  ],
+  "license": "ISC",
```

Is ISC what we're going with these days? My understanding was we were using CDL for core holochain things, and Apache 2.0 for more permissive things we'd like to spread wider.

---

ISC is just NPM's default, we can change it to CDL.
```suggestion
  "license": "CDL",
```

---

Lair itself is under the more permissive Apache-2.0.

---

```suggestion
  "license": "Apache-2.0",
```